CppSamples/DisplayInformation/ShowGrid/README.metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-    "category": "Display information",
-    "description": "Display coordinate system grids including Latitude/Longitude, MGRS, UTM and USNG on a map view. Also, toggle label visibility and change the color of grid lines and grid labels.",
+    "category": "Show grid",
```

Comment:
Is there a new category for `Show grid` ?

Reply:
Yeah, I'm not sure why the category got removed, sorry

Reply:
You're right, I confused myself with the component complete changes